GameData/KerbalismConfig/Support/ROCapsules.cfg

```diff
@@ -601,26 +1002,47 @@
 {
 	@MODULE[Habitat]
 	{
-		%volume = 8.95
-		%surface = 35
+		%volume = 9	//m^3
+		%surface = 4.3267	//m^2
+		%max_pressure = 0.35
 	}
 }
 @PART[ROC-CSTCM]:NEEDS[ROCapsules,FeatureHabitat]:AFTER[zzzKerbalism]
 {
 	@MODULE[Habitat]
 	{
-		%volume = 11
-		%surface = 35	//guess, very similar dimensions to Orion
+		%volume = 11	//m^3
+		%surface = 4.9461	//m^2
+		%max_pressure = 0.35
 	}
 }
+@PART[ROC-Dragon2CM|ROC-Dragon2CMLunar]:NEEDS[ROCapsules,FeatureHabitat]:AFTER[zzzKerbalism]
+{
+	@MODULE[Habitat]
+	{
+		%volume = 11.8064	// m^3 ~ Calculated from the Internal Model Volume
+		%surface = 5.185	// m^2 ~ Calculated from the Internal Model Volume
+		%max_pressure = 0.35
+	}
+}
+@PART[ROC-StarshipLunarCM]:NEEDS[ROCapsules,FeatureHabitat]:AFTER[zzzKerbalism]
+{
+	@MODULE[Habitat]
+	{
+		%volume = 297.531	// m^3 ~ Calculated from the Internal Model Volume
+		%surface = 44.5678	// m^2 ~ Calculated from the Internal Model Volume
+		%max_pressure = 0.35
+	}
+}
+
 // ============================================================================
 // SCIENCE
 // ============================================================================
 // crew experiments
 
 @PART[ROC-MercuryCM|ROC-MercuryCMBDB|ROC-VostokCapsule]:NEEDS[ROCapsules,FeatureScience,RP-0]:BEFORE[RP-0-Kerbalism] { %capsuleTier = Basic }
-@PART[ROC-GeminiCM|ROC-GeminiCMBDB|ROC-VoskhodCapsule|ROC-DynaBody]:NEEDS[ROCapsules,FeatureScience,RP-0]:BEFORE[RP-0-Kerbalism]{ %capsuleTier = SecondGen }
-@PART[ROC-ApolloCM|ROC-ApolloCMBlockIII|ROC-OrionCM]:NEEDS[ROCapsules,FeatureScience,RP-0]:BEFORE[RP-0-Kerbalism] { %capsuleTier = Mature }
+@PART[ROC-GeminiCM|ROC-GeminiCMBDB|ROC-D2MissionModule1|ROC-VoskhodCapsule|ROC-DynaBody]:NEEDS[ROCapsules,FeatureScience,RP-0]:BEFORE[RP-0-Kerbalism]{ %capsuleTier = SecondGen }
+@PART[ROC-ApolloCM|ROC-D2MissionModule2|ROC-ApolloCMBlockIII|ROC-OrionCM|ROC-Dragon2CM|ROC-Dragon2CMLunar|ROC-StarshipLunarCM]:NEEDS[ROCapsules,FeatureScience,RP-0]:BEFORE[RP-0-Kerbalism] { %capsuleTier = Mature }
```

Comment:
I wouldn't refrain from giving StarshipLunar a Lab module, it's basically a space station on its own